src/AdminPage/components/EmailModal.js

```diff
@@ -21,17 +21,20 @@ class EmailModal extends Component {
   handleClose() {
     this.setState({ open: false });
   }
-
+  
```

Comment:
I think we want to try to avoid tabbed but otherwise empty lines, but it's not a big deal.